.changes/fix-use-correct-sync-options.md

```diff
@@ -0,0 +1,6 @@
+
+---
+"client-nodejs-binding": patch
```

Comment:
```suggestion
"wallet-nodejs-binding": patch
```

Reply:
Whoops, didn't notice that. Thanks a lot!